frontend/src/views/ComplianceReports/components/AssessmentCard.jsx

```diff
@@ -4,70 +4,245 @@ import { List, ListItem, ListItemText, Stack, Typography } from '@mui/material'
 import BCWidgetCard from '@/components/BCWidgetCard/BCWidgetCard'
 import { timezoneFormatter } from '@/utils/formatters'
 import { COMPLIANCE_REPORT_STATUSES } from '@/constants/statuses'
+import { constructAddress } from '@/utils/constructAddress'
+import BCButton from '@/components/BCButton'
+import AssignmentIcon from '@mui/icons-material/Assignment'
+import { useCreateSupplementalReport } from '@/hooks/useComplianceReports'
+import Box from '@mui/material/Box'
+import { useNavigate } from 'react-router-dom'
+import { StyledListItem } from '@/components/StyledListItem'
+import { roles } from '@/constants/roles'
+import { Role } from '@/components/Role'
 
-export const AssessmentCard = ({ orgName, hasMet, history, isGovernmentUser }) => {
+export const AssessmentCard = ({
+  orgData,
+  hasMet,
+  history,
+  isGovernmentUser,
+  currentStatus,
+  complianceReportId,
+  alertRef
+}) => {
   const { t } = useTranslation(['report'])
+  const navigate = useNavigate()
+
   const filteredHistory = useMemo(() => {
     if (!history || history.length === 0) {
       return []
     }
     // Sort the history array by date in descending order
-    return [...history].sort((a, b) => {
-      return new Date(b.createDate) - new Date(a.createDate)
-    }).map(item => {
-      if (item.status.status === COMPLIANCE_REPORT_STATUSES.ASSESSED && !isGovernmentUser) {
-        item.status.status = "AssessedBy"
-      }
-      return item
-    }).filter(item => item.status.status !== COMPLIANCE_REPORT_STATUSES.DRAFT)
+    return [...history]
+      .sort((a, b) => {
+        return new Date(b.createDate) - new Date(a.createDate)
+      })
+      .map((item) => {
```

Comment:
I noticed few things during my local testing. 

- Looks like the new history list expects "Compliance Report:" to be prefixed before every history record.
- When new supplemental report is submitted, assessment card does not show up.
- May need to implement the ability for the user to expand or collapse full history. (I'm not quite sure, but this might be altogether a new ticket).

Reply:
Confirmed with Alex that updating history will be its own ticket. Good catch on the issue with supplementals, added a new flag and unit test to cover this case.